packages/auth/src/tests/app.ts

```diff
@@ -18,7 +30,7 @@ export const createTestApp = async (
   container: IocContract<AppServices>
 ): Promise<TestContainer> => {
   const config = await container.use('config')
-  config.port = 0
+  config.authPort = 0
```

Comment:
```suggestion
  config.adminPort = 0
```
Otherwise, I get this error running tests:
```
    listen EADDRINUSE: address already in use :::3003

      186 |     )
      187 |
    > 188 |     this.adminServer = httpServer.listen(port)
          |                                   ^
      189 |   }
      190 |
      191 |   public async startAuthServer(port: number | string): Promise<void> {

      at App.listen [as startAdminServer] (packages/auth/src/app.ts:188:35)
```